content/docs/alerting/configuration.md

```diff
@@ -376,7 +376,7 @@ channel: <tmpl_string>
 ```
 
 
-## OpsGenie receiver `<opsgenie_config>`
```

Comment:
The receiver block is correctly specified here: https://prometheus.io/docs/alerting/configuration/#receiver-receiver

You see `<opsgenie_config>` in there, which is meant as a reference defined in this block.